regression/strings/smoke_tests/performance.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python
+import time
+from subprocess import check_output
+
+git_output=check_output(["git", "show", "HEAD"])
+commit=git_output.split('\n')[0]
+
+commit_id=time.strftime("%Y_%m_%d__%H_%M_%S")+"__"+commit[7:]
+
+process = check_output(["grep", "^Runtime\ decision\ procedure", "-R"])
+
+file_name='performance_'+commit_id+'.out'
+print "writing to file", file_name
+f=open(file_name, 'w')
+
+for x in process.split('\n'):
```

Comment:
I'm not sure what you mean here. I can't grep on the folder where `f` is currently being written, I need to buffer somehow. But I removed the `split` which is unnecessary.